housekeeper/store/api/crud/update.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from housekeeper.store.api.base import BaseHandler
+from housekeeper.store.models import Bundle, File, Version, Tag
+from typing import Dict, List, Tuple
+from pathlib import Path
+
+LOG = logging.getLogger(__name__)
+
+
+class UpdateHandler(BaseHandler):
+    """This class handles updating model entries in the store"""
+
+    def update_bundle_with_version(
+        self,
+        data: dict,
+        bundle: Bundle,
+    ) -> Version:
+        """Build a new version object and add it to an existing bundle"""
+        created_at = data.get("created_at", data.get("created"))
+        if self.get_version_by_date_and_bundle_name(
+            version_date=created_at, bundle_name=bundle.name
+        ):
+            LOG.info("version of bundle already added")
+            return None
+
+        version_obj = self.create_version(
+            created_at=created_at,
+            expires_at=data.get("expires_at", data.get("expires")),
```

Comment:
```suggestion
            created_at=created_at
            expires_at=data.get("expires_at", data.get("expires")),
```
Or? Do you know why the comma is there?

Reply:
I think BLACK likes to do that.